### DISCUSSION TITLE
increasing setting  of bucket folder

### DISCUSSION THREAD

**Discussion by ydcteam** [remotely-save/remotely-save#318] [Ideas]
HI, First appreciating you bring convenience  through this plugin.....

Could you pls increase setting of  bukcket folder? Inside same  bucket, there are different repo which is matched  to relevant repo of obsidian?? In this  case I don't need to use  too many buckets.....

Tks a lot for reply。。。。
Don

**Comment by fyears** [comment]
latest version supports this
